pkg/sql/logictest/testdata/logic_test/show_trace

```diff
@@ -13,10 +13,10 @@ query TTT
 SELECT span, message, operation FROM [SHOW TRACE FOR SESSION] WHERE message NOT LIKE e'%\n%' AND message NOT LIKE '%canceling context'
 ----
 (0,0)  === SPAN START: sql txn implicit ===  sql txn implicit
-(1,0)  === SPAN START: sql txn ===           sql txn implicit
-(1,0)  executing 1/5: BEGIN TRANSACTION      sql txn implicit
-(1,0)  executing 2/5: SELECT 1               sql txn implicit
-(1,0)  executing 3/5: COMMIT TRANSACTION     sql txn implicit
+(1,0)  === SPAN START: sql txn ===           sql txn
```

Comment:
I hadn't seen this part of the patch. When I reviewed the PR I saw a TC failure and I thought this was a flaky test. I don't understand why this needs to change.

Reply:
I didn't realize when I first made the change that `SHOW TRACE` [uses window functions](https://github.com/cockroachdb/cockroach/blob/6dc8f306719a0be1a0627fe3419315f73643b740/pkg/sql/show.go#L294) under the hood. This trace was actually incorrect before because of the bug this change fixes.

Reply:
Hah! Amazing. Figure, I wrote that code, I simply had forgotten. :) Carry on.